### PR TITLE
docs: translate `renderToReadableStream` section

### LINKED ISSUE #459: `renderToReadableStream`

### PR DESCRIPTION
**Pull request by hisamafahri**
Close #459

### CODE REVIEW

**Comment by mazipan** [comment]
Is this ready to review @hisamafahri?

**Comment by hisamafahri** [comment]
@mazipan not yet. This section's quite a lot.

**Comment by hisamafahri** [comment]
@mazipan this is ready for review now 👍

**Comment by resir014** [comment]
Hi @hisamafahri, there's a merge conflict. Please resolve it first before proceeding with review.

File: src/content/reference/react-dom/server/renderToReadableStream.md [hunk 1]
```diff
@@ -41,44 +41,44 @@ async function handler(request) {
...
 
-The returned stream has an additional property:
+_Stream_ yang dikembalikan memiliki properti tambahan:
 
-* `allReady`: A Promise that resolves when all rendering is complete, including both the [shell](#specifying-what-goes-into-the-shell) and all additional [content.](#streaming-more-content-as-it-loads) You can `await stream.allReady` before returning a response [for crawlers and static generation.](#waiting-for-all-content-to-load-for-crawlers-and-static-generation) If you do that, you won't get any progressive loading. The stream will contain the final HTML.
+* `allReady`: _Promise_ yang diselesaikan saat semua preose _render_ selesai, termasuk [_shell_](#specifying-what-goes-into-the-shell) dan semua [konten](#streaming-more-content-as-it-loads) tambahan. Anda dapat menggunakan`await stream.allReady` sebelum mengembalikan respons [untuk _crawler_ dan _static generation_.](#waiting-for-all-content-to-load-for-crawlers-and-static-generation) Jika Anda melakukannya, Anda tidak akan mendapatkan pemuatan progresif. _Stream_ akan berisi HTML final.
```
Comment (mazipan) [inline src/content/reference/react-dom/server/renderToReadableStream.md:73]: Pake `*` buat bikin italic, biar konsisten dengan halaman lainnya

File: src/content/reference/react-dom/server/renderToReadableStream.md [hunk 2]
```diff
@@ -368,13 +368,13 @@ async function handler(request) {
 }
 ```
 
-If you provide a custom `onError` implementation, don't forget to also log errors to the console like above.
+Jika Anda memberikan implementasi `onError` khusus, jangan lupa juga mencatat kesalahan ke konsol seperti di atas.
 
 ---
 
-### Recovering from errors inside the shell {/*recovering-from-errors-inside-the-shell*/}
+### Memulihkan dari kesalahan di dalam *shell* {/*recovering-from-errors-inside-the-shell*/}
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToReadableStream.md:375]: Italic tak perlu ada di Heading

File: src/content/reference/react-dom/server/renderToReadableStream.md [hunk 3]
```diff
@@ -413,13 +413,13 @@ async function handler(request) {
 }
 ```
 
-If there is an error while generating the shell, both `onError` and your `catch` block will fire. Use `onError` for error reporting and use the `catch` block to send the fallback HTML document. Your fallback HTML does not have to be an error page. Instead, you may include an alternative shell that renders your app on the client only.
+Jika ada kesalahan saat membuat *shell*, `onError` dan blok `catch` Anda akan aktif. Gunakan `onError` untuk pelaporan kesalahan dan gunakan blok `catch` untuk mengirim dokumen HTML *fallback*. HTML *fallback* Anda tidak harus berupa halaman kesalahan. Sebagai gantinya, Anda dapat menyertakan *shell* alternatif yang merender aplikasi Anda hanya pada klien.
 
 ---
 
-### Recovering from errors outside the shell {/*recovering-from-errors-outside-the-shell*/}
+### Memulihkan dari kesalahan di luar *shell* {/*recovering-from-errors-outside-the-shell*/}
```
Comment (r17x) [inline src/content/reference/react-dom/server/renderToReadableStream.md:420]: idem

**Review by r17x — CHANGES_REQUESTED** [review]
Italic tak perlu ada di Heading

**Review by r17x — APPROVED** [review]

**Pull request merged**